lib/chef/knife/vsphere_vm_snapshot.rb

```diff
@@ -142,12 +147,13 @@ def find_node(tree, name)
   end
 
   def display_node(node, current, shift = 1)
+    descr = node.name + ' ' + node.createTime.iso8601
     out = ''
     out << '+--' * shift
     if node.snapshot == current
-      out << "#{ui.color(node.name, :cyan)}" << "\n"
+      out << ui.color(descr, :cyan) << "\n"
```

Comment:
Rubocop will probably complain about the double quotes around the `\n`, and the line below.

Reply:
yep I can change this as well.